rust/src/fvm/externs.rs

```diff
@@ -27,7 +26,7 @@ impl CgoExterns {
 impl Rand for CgoExterns {
     fn get_chain_randomness(
         &self,
-        pers: DomainSeparationTag,
+        pers: i64,
```

Comment:
why this change?

Reply:
This is mostly a change to enable M2, when users will want to specify their own. Basically, the actors care about these tags, but nobody else does.